### DISCUSSION TITLE
Read Excel in Svelte (images, cell colors, comments)

### DISCUSSION THREAD

**Discussion by Amerlander** [exceljs/exceljs#2643] [Show and tell]
I played around in a Svelte REPL how to read Excel files. So I thought I just share my result: https://svelte.dev/repl/f608ee6ceb3c4b5590ec9f1bb0fe42e8?version=4.2.8

- Images are read and attached to the row. They are not attached to the Cell, since I further process the data and therefore didn't bother about whether it is in the right cell.
- Cell color is read and converted to rgba()
- Cell comment text and author is read

Feedback is welcome!

![image](https://github.com/exceljs/exceljs/assets/3764089/9be06487-ea3d-4a62-958a-d667cf81bf6d)

![image](https://github.com/exceljs/exceljs/assets/3764089/8cd33f23-12a8-483a-ae44-14f72a6faf36)

**Comment by manutheblacker** [comment]
Thank you for sharing this tutorial.
